google-oauth-client/src/test/java/com/google/api/client/auth/oauth/OAuthHmacSignerTest.java

```diff
@@ -32,4 +35,39 @@ public void testComputeSignature() throws GeneralSecurityException {
     signer.tokenSharedSecret = "def";
     assertEquals(EXPECTED_SIGNATURE, signer.computeSignature("foo"));
   }
+
+  public void testComputeSignatureHmacSha256() throws GeneralSecurityException {
+    final OAuthHmacSigner signer = new OAuthHmacSigner();
```

Comment:
Google style specifies no final on local variables except where required

Reply:
this has been reverted back in favor of a new class